linchpin/InventoryFilters/JSONInventoryFormatter.py

```diff
@@ -73,7 +73,7 @@ def add_ips_to_groups(self, inven_hosts, layout):
                         self.config["all"]["hosts"].append(ip)
         return True
 
-    def add_common_vars(self, host_groups, layout):
+    def add_common_vars(self, host_groups, layout, config):
```

Comment:
I see a new argument added here but I don't see any references to the code that calls this.

Reply:
The code that calls it was updated in #703.  It's the same as the code that calls this method in `CFGInventoryFormatter.py`